esmvaltool/recipes/hydrology/recipe_globwat.yml

```diff
@@ -0,0 +1,98 @@
+# ESMValTool
+# globwat.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim and ERA-5 data
+    for use in the GlobWat hydrological model.
+    GlobWat is a Global Hydrological Model provided by FAO
+    http://www.fao.org/nr/water/aquamaps/
+
+    By default, PET is calculated by the De Bruin method. However, users
+    can choose the Langbein method by setting the langbein_pet in the recipe
+    to True.
+
+    More information about the De Bruin method can be found at:
+    https://doi.org/10.1175/JHM-D-15-0006.1
+    More information about the Langbein method can be found at:
+    https://doi.org/10.3133/cir52
+
+    The target file has been generated from one of the models sample files
+    (prc01wb.asc). The file can be found at:
+    http://www.fao.org/geonetwork/srv/en/metadata.show?currTab=simple&id=52366
+
+    Regridding scheme can be choosen from ‘linear’, ‘linear_extrapolate’,
+    ‘nearest’, ‘area_weighted’, and ‘unstructured_nearest’.
+
+    The rough cutout preprocessor can be used by users to process the data
+    for their desired region. The data will be processed at the global scale 
+    if the preprocessor in the recipe is commented.
+
+  authors:
+    - abdollahi_banafsheh
+    - alidoost_sarah
+
+  projects:
+    - ewatercycle
+
+  references:
+    - acknow_project
+
+preprocessors:
+    rough_cutout:
+      extract_region:
+        start_longitude: 40
+        end_longitude: 65
+        start_latitude: 25
+        end_latitude: 40
+
+diagnostics:
+  diagnostic_daily_GlobWat:
+    description: daily precipitation of ERA5 & ERA-Interim
+    additional_datasets:
+      - {dataset: ERA-Interim, project: OBS6, tier: 3, type: reanaly, version: 1}
+      - {dataset: ERA5, project: OBS6, type: reanaly, version: '1', tier: 3}
+    variables:
+      pr: &daily_var
+        mip: day
+        start_year: 1986
+        end_year: 2016
+        # comment preprocessor to process data on the global scale
+        preprocessor: rough_cutout
+      tas: *daily_var
+      # comment psl, rsds and rsdt if langbein_pet is True
+      psl: *daily_var
+      rsds: *daily_var
+      rsdt:
+        <<: *daily_var
+        mip: CFday
+    scripts:
+      script:
+        script: hydrology/globwat.py
+        target_file: 'globwat/globwat_target_grid.nc'
+        langbein_pet: False
+        regrid_scheme: linear
```

Comment:
You might want to consider area-weighted (first order conservative) regridding for precipitation. This ensures that the total volume of water is consistent before and after regridding.

Reply:
Area-weighted added.